runtime/cpp/yolov5_yolov7_detection/CMakeLists.txt

```diff
@@ -20,16 +20,15 @@ find_package( OpenCV REQUIRED)
 message(STATUS "Found OpenCV: " ${OpenCV_INCLUDE_DIRS})
 
 file(GLOB SOURCES
-    double_buffer.cpp
-    yolo_output.cpp
-    yolo_postprocess.cpp
-    yolov5_yolov7_inference.cpp
+    ./*.cpp
+    ./common/*.cpp
```

Comment:
a. It's better to write the files explicitly.
b. are you sure ./common/*.cpp should all be compiled as source code for this project? It made a compilation issue last time.